hooks/shapediver/parameters/useParameterImportExport.ts

```diff
@@ -0,0 +1,162 @@
+
+	/**
+	 * Export parameters as JSON file
+	 */
+	const exportParameters = useCallback(async () => {
+		try {
+			// Get current parameter values from store
+			const parameters = useShapeDiverStoreParameters
+				.getState()
+				.getParameters(namespace);
+			const parameterValues: {[key: string]: any} = {};
+
+			Object.values(parameters).forEach((paramStore) => {
+				const param = paramStore.getState();
+				parameterValues[param.definition.id] = param.state.uiValue;
+			});
+
+			// Create JSON blob and download
+			const jsonContent = JSON.stringify(parameterValues, null, 2);
+			const blob = new Blob([jsonContent], {type: "application/json"});
+			const url = URL.createObjectURL(blob);
+
+			// Create download link
+			const link = document.createElement("a");
+			link.href = url;
+			link.download = `parameters_${namespace}_${new Date().toISOString().split("T")[0]}.json`;
+			document.body.appendChild(link);
+			link.click();
+			document.body.removeChild(link);
+			URL.revokeObjectURL(url);
+
+			notifications.success({
+				message: "Parameters exported successfully",
+			});
+		} catch (error) {
+			console.error("Error exporting parameters:", error);
+			notifications.error({
+				message: "Failed to export parameters",
+			});
+		}
+	}, [namespace]);
+
+	/**
+	 * Import parameters from JSON file
+	 */
+	const importParameters = useCallback(async () => {
+		return new Promise<void>((resolve, reject) => {
+			const fileInput = document.createElement("input");
+			fileInput.type = "file";
+			fileInput.accept = ".json";
+
+			fileInput.onchange = async (event: Event) => {
+				const target = event.target as HTMLInputElement;
+				const file = target.files?.[0];
+
+				if (!file) {
+					reject(new Error("No file selected"));
+					return;
+				}
+
+				try {
+					const text = await file.text();
+					const parameterValues = JSON.parse(text);
+
+					// Get current parameters from store for validation
+					const sessionParameters = useShapeDiverStoreParameters
+						.getState()
+						.getParameters(namespace);
+					const validParameters: {[key: string]: any} = {};
+
+					Object.keys(parameterValues).forEach((paramId) => {
+						if (paramId in sessionParameters) {
+							validParameters[paramId] = parameterValues[paramId];
+						} else {
+							console.warn(
+								`Parameter ${paramId} not found in session, skipping`,
+							);
+						}
+					});
+
+					if (Object.keys(validParameters).length === 0) {
+						throw new Error("No valid parameters found in file");
+					}
+
+					// Apply parameter values
+					await batchParameterValueUpdate(namespace, validParameters);
+
+					notifications.success({
+						message: "Parameters imported successfully",
+					});
+					resolve();
+				} catch (error) {
+					console.error("Error importing parameters:", error);
+					notifications.error({
+						message:
+							"Failed to import parameters: " +
+							(error as Error).message,
+					});
+					reject(error);
+				}
```

Comment:
@LemToUp related to this [comment](https://github.com/shapediver/AppBuilderShared/pull/118/files#r2171041826):

  * Parameters should be searched first by id, then by name
  * In case some of the parameters could not be found, we should show a warning notification listing all the names of the parameters that could not be found. 
  * In case all of the parameters could not be found, show an error notification. 
  * Let's not use try-catch here, because we want errors to be reported to sentry

Reply:
Done: 59b8e1d93fb87c794eaa1a9ce41ba25379cd71e0